src/main/frontend/ui.cljs

```diff
@@ -716,13 +720,26 @@
             (modal-panel show? modal-panel-content state close-fn false close-btn?)))]))))
 
 (defn loading
+  ([] (loading (t :loading)))
   ([content] (loading content nil))
   ([content opts]
    [:div.flex.flex-row.items-center.inline
     [:span.icon.flex.items-center (svg/loader-fn opts)
      (when-not (string/blank? content)
        [:span.text.pl-2 content])]]))
 
+(defn notify-graph-persist!
```

Comment:
At some point it could be nice if non-components fns moved to another ns as this ns seems like it's outgrowing its primary purpose of providing reusable components